src/main/java/com/shipmentEvents/handlers/EventHandler.java

```diff
@@ -97,7 +97,7 @@ private void processShipmentUpdates(final LambdaLogger logger) throws Interrupte
         final AmazonS3 s3Client = EventHandler.getS3Client();
         logger.log("Processing Bucket: " + bucketName);
 
-        ObjectListing files = s3Client.listObjects(bucketName);
+        ObjectListing files = s3Client.listObjectsV2(bucketName);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code might not produce accurate results if the operation returns paginated results instead of all results. Consider adding another call to check for additional results.